cmd/cli/uninstall_mesh.go

```diff
@@ -179,6 +179,18 @@ func (d *uninstallMeshCmd) run() error {
 			}
 
 			if err == nil {
+				namespaces, _ := selectNamespacesMonitoredByMesh(m.name, d.clientSet)
+				for _, ns := range namespaces.Items {
+					removeCmd := &namespaceRemoveCmd{
+						out:       d.out,
+						namespace: ns.Name,
+						meshName:  m.name,
+						clientSet: d.clientSet,
+					}
+
+					_ = removeCmd.run()
+
```

Comment:
To resolve this lint error we can remove the newline on  line192.

Reply:
Thanks, @jaellio for the review.